### PR TITLE
Mailbox 267 change unstable test about nextModSeq

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

**Comment by rouazana** [comment]
Please check indent on some files.

**Comment by chibenwa** [comment]
:+1:

**Comment by aduprat** [comment]
:+1:

File: mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraUidAndModSeqProviderTest.java [hunk 1]
```diff
@@ -49,79 +49,47 @@
...
     @Test
     public void lastUidShouldRetrieveValueStoredByNextUid() throws Exception {
-        MailboxPath path = new MailboxPath("gsoc", "ieugen", "Trash");
-        SimpleMailbox<CassandraId> newBox = new SimpleMailbox<>(path, 1234);
-        mapper.save(newBox);
-        mailboxList.add(newBox);
-        pathsList.add(path);
-
-        long result = uidProvider.lastUid(null, newBox);
+    	int nbEntries = 100;
```
Comment (rouazana) [inline mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraUidAndModSeqProviderTest.java]: indent: here fox example

**Comment by mbaechler** [comment]
ok, checked with `for i in `git show --name-only  |tail  -n +7`; do grep -lP '\t' $i; done`
I found two files with some tabs.

**Comment by rouazana** [comment]
:+1:

**Pull request merged**